examples/http-certification/assets/src/backend/src/lib.rs

```diff
@@ -71,6 +93,30 @@ fn certify_all_assets() {
             }],
             aliased_by: vec!["/".to_string()],
         },
+        AssetConfig::File {
+            path: "index.html.gzip".to_string(),
+            content_type: Some("text/html".to_string()),
+            headers: get_asset_headers(vec![(
+                "cache-control".to_string(),
+                "public, no-cache, no-store".to_string(),
+            )]),
+            fallback_for: vec![AssetFallbackConfig {
+                scope: "/".to_string(),
+            }],
+            aliased_by: vec!["/".to_string()],
+        },
+        AssetConfig::File {
+            path: "index.html.br".to_string(),
```

Comment:
one alternative could be to have path be an enum that is either a specific file because it's only provided for one encoding or many, something like:

```
// for a single encoding
AssetConfig::File {
   path: AssetPath::File("index.html".to_string())
}

// for many encoding
AssetConfig::File {
   path: AssetPath::Alternate(vec![
     AssetFileEncoded::new("index.html".to_string(), Encoding::Identity)),
     AssetFileEncoded::new("index.html.gzip".to_string(), Encoding::Gzip)),
     AssetFileEncoded::new("index.html.br".to_string(), Encoding::Brotli)
   ])
}
```

or if you would want to make it strongly typed to avoid misconfigurations of adding to many times the same encoding:

```
AssetConfig::File {
   path: AssetPath::new("index.html".to_string())
     .with_gzip("index.html.gzip".to_string())
     .with_brotli("index.html.br".to_string())
     .build()
}
```